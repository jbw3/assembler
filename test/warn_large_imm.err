WARNING: line: 3, col: 10
Immediate value "300" was truncated.
