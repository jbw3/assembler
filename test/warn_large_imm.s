# Warning if an immediate value will be truncated
mov  r0, r1
addi r1, 300
mov  r2, r3
addi r2, 5
