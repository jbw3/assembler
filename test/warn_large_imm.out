0101
b12c
0123
b205
